api/v1/types.go

```diff
@@ -157,6 +157,19 @@ type ComponentInfo struct {
 
 type GPUdComponentInfos []ComponentInfo
 
+type PackageStatus struct {
+	Name           string        `json:"name"`
+	IsInstalled    bool          `json:"is_installed"`
+	Installing     bool          `json:"installing"`
+	Progress       int           `json:"progress"`
+	TotalTime      time.Duration `json:"total_time"`
+	Status         bool          `json:"status"`
+	TargetVersion  string        `json:"target_version"`
```

Comment:
and why do we report the target version?

Reply:
removed.